lib/raidenclient/RaidenClient.ts

```diff
@@ -96,26 +97,21 @@ class RaidenClient extends BaseClient {
 
     this.port = port;
     this.host = host;
+    this.disable = disable;
```

Comment:
Looking at the `LndClient` it might make sense to also keep port, host, disable etc. in `private config`. Probably a bit out of scope for this PR, but something to think about.

Reply:
I was thinking about that too. I'll go ahead ane make the change.